libs/jitsu-js/src/analytics-plugin.ts

```diff
@@ -445,6 +446,7 @@ function adjustPayload(
   delete payload.properties?.context;
   const referrer = runtime.referrer();
   const context: AnalyticsClientEvent["context"] = {
+    ...config.defaultPayloadContext,
```

Comment:
it is not possible to introduce nested property like `context.page.awesomePageProp` this way.

Better to use `deepMerge` approach one the line 484 bellow

Reply:
Yeah, agreed. I'll look into it.